pkgs/applications/video/kodi/addons/youtube/default.nix

```diff
@@ -23,15 +23,6 @@ buildKodiAddon rec {
     };
   };
 
-  patches = [
-    # This patch can be removed once https://github.com/anxdpanic/plugin.video.youtube/pull/260 has been merged.
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Oops! Done.